examples/ex5/ex5.cpp

```diff
@@ -14,6 +14,7 @@ double CenterCoordinate(int i, int coor);
 
 /* Global Variables/Parameters */
 double Time;
+double *stepTime = (double *)calloc(1000,sizeof(double));
```

Comment:
1000 may be adhoc, allocate using nPlotSteps.

Reply:
good point.